src/util/system.cpp

```diff
@@ -1186,6 +1211,44 @@ void RenameThread(const char* name)
 #endif
 }
 
+void SpectreMitigation(bool enable)
+{
+#if defined(HAVE_SYS_PRCTL_H)
```

Comment:
Please add a common prefix to these log messages to be able to quickly grep for it. E.g. `SpectreMitigation:` using `%s:` and `__func__`. It also makes it seem less to come out of the blue for users.